mitype/app.py

```diff
@@ -297,7 +308,10 @@ def update_state(self, win):
         win.addstr(self.line_count, 0, " " * self.window_width)
         win.addstr(self.line_count + 2, 0, " " * self.window_width)
         win.addstr(self.line_count + 4, 0, " " * self.window_width)
-        win.addstr(self.line_count, 0, self.current_word)
+        if self.current_word_limit_reached:
```

Comment:
Can we just use -
`if len(self.current_word) >= self.current_word_limit:`
and avoid the usage of `self.current_word_limit_reached` totally?

Reply:
This will also solve the issue of having to reset this variable back to false, as currently if you reach the limit and then you hit ESC to reset the test, the text would still appear red.

Reply:
Got it!